raiden/transfer/state.py

```diff
@@ -459,6 +459,7 @@ class TokenNetworkState(State):
     partneraddresses_to_channelidentifiers: Dict[Address, List[ChannelID]] = field(
         repr=False, default_factory=lambda: defaultdict(list)
     )
+    fee_schedule: Optional[FeeScheduleState] = field(repr=False, default=None)
```

Comment:
What's the rational behind adding this here as well as having it in the channel objects? When is this value used instead of the channels fee schedule?